src/components/Nav/Search/Search.js

```diff
@@ -127,13 +106,7 @@ class Search extends Component {
         />
         <div className={isListActive ? 'listBox' : 'displayNone'}>
           <div className='searchList'>
-            <div className='searchHeaderWrapper'>
-              <span>최근 검색어</span>
-              <div className='keywordDeleteBtn' onClick={this.deleteKeywords}>
-                모두 삭제
-              </div>
-            </div>
-            <ul className='latestList'>
+            <ul className='filteredList'>
               {filteredMovie?.length > 0 &&
                 filteredMovie.map((movie) => (
```

Comment:
둘 중 하나만 선택해서 사용해주시면 됩니다.
```suggestion
              {filteredMovie.length > 0 &&
                filteredMovie.map((movie) => (
```
혹은
```suggestion
              {filteredMovie?.map((movie) => (
```
다만 옵셔널 체이닝 같은 경우 es7 문법을 지원하지 않는 환경에서는 작동하지 않을 수 있습니다.

Reply:
후자의 경우에는 map이 undefined라는 오류가 떠서 전자로 고쳐놨습니다. 감사합니다! :)